src/test/integration/suite/test-utils.ts

```diff
@@ -36,3 +46,78 @@ export const getEnvironmentMock = (
   },
   ...overrides,
 });
+
+export enum MessageType {
+  INFORMATION = "showInformationMessage",
+  ERROR = "showErrorMessage",
+  WARNING = "showWarningMessage",
+}
+
+export const stubShowMessage = (messageType: MessageType) => {
+  const openExternalMock = jestMock.spyOn(vscode.env, "openExternal");
+  const mockUriParse = jestMock
+    .spyOn(vscode.Uri, "parse")
+    .mockImplementation((url) => url as any);
+  openExternalMock.mockResolvedValue(true);
+  const showMessageMock = jestMock.spyOn(vscode.window, messageType);
+  showMessageMock.mockResolvedValue("More info" as any);
```

Comment:
@meniRoy I'm not sure I'm following what's going on here.
you mock `showMessageMock.mockResolvedValue("More info" as any);`

then you verify that it was called with "more info"? 🤔

Reply:
9bc46ec54402604d0b203a9d270e72d977906c35